package.json

```diff
@@ -69,13 +69,11 @@
     "class-is": "^1.1.0",
     "multicodec": "^1.0.1",
     "multihashing-async": "~0.8.1",
-    "protons": "^1.0.2",
+    "protons": "git://github.com/gozala/protons#uint8array",
```

Comment:
I planned to switch to https://github.com/mapbox/pbf if we ever switch to UInt8Array support. It might be worth a look now. IIRC that version just works automatically on any array-like thing, hence Node.js Buffers as well as UInt8Arrays.

Reply:
That sounds like a good idea! However I would much rather keep the rabbit hole as shallow as possible. That is to say can we do that independently of this pull request ?

P.S.: I obviously don't suggest landing this with my fork in dependencies, just have it here until this https://github.com/dignifiedquire/borc/pull/48 makes it's way to npm.